src/nix/host/ssh.rs

```diff
@@ -345,6 +345,8 @@ impl Ssh {
             "-o",
             "BatchMode=yes",
             "-T",
+            "-o", "ControlMaster=no",
+            "-o", "ControlPath=/var/empty/non-existant",
```

Comment:
Why do you bother setting `ControlPath`? If `ControlMaster` is `no`, `ssh` won't even look at `ControlPath`.